get-logs.py

```diff
@@ -23,6 +23,12 @@ def parse_args():
         action="store_true",
         help="Try to explain the log using log-detective.com",
     )
+    parser.add_argument(
+    "-l",
+    "--local",
+    action="store_true",
+    help="Try to explain the log using a local log-detective model"
```

Comment:
Nitpick: We should follow the same indent here too:
```suggestion
        "-l",
        "--local",
        action="store_true",
        help="Try to explain the log using a local log-detective model"
```